api/src/main/java/org/motechproject/nms/api/web/contract/FrontLineWorkerUser.java

```diff
@@ -7,8 +7,8 @@
     private long currentUsageInPulses;
     private long endOfUsagePromptCounter;
```

Comment:
did we standardize on the primitive type vs. object types (Long). I would think that we would use Long for contract serialization?